### PR TITLE
Add email for competition sign-up form

### PR DESCRIPTION
**Pull request by Adimote**

### CODE REVIEW

File: SR2024/2024-01-24-competition-sign-up-form.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+Teams have been diligently building their robots for 6 months to match their wits against other teams at this grand finale, - and we need **you** to make it happen! 
+
+We try hard to make this event as exciting as possible. Not only for the competitors, but also the volunteers, too. We'll ensure you have a fulfilling and memorable weekend, without causing too much stress. 
+
+Please share this sign-up form with all of your friends and family. The more people we have sign up, the smoother it will run.
+
+If you're interested in attending, please sign up with the link below:
+
+[**Sign up to volunteer**](<FORM GOES HERE>)
```
Comment (prophile) [inline SR2024/2024-01-24-competition-sign-up-form.md:20]: We should put a link here

File: SR2024/2024-01-24-competition-sign-up-form.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+---
+to: All Volunteers
+subject: Sign up to volunteer at SR2024
+---
+
+Hi Everyone, 
+
+We've opened Volunteer sign-ups for the competition weekend of SR2024!
```
Comment (prophile) [inline SR2024/2024-01-24-competition-sign-up-form.md:8]: Do we want this capitalised?

File: SR2024/2024-01-24-competition-sign-up-form.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+The competition will take place at the University of Southampton on April 13th - 14th, with optional set-up days on the 11th & 12th. We'd love it if you could volunteer some of your time to help things run smoothly!
```
Comment (prophile) [inline SR2024/2024-01-24-competition-sign-up-form.md:10]: We might want to use either & for both or – for both

File: SR2024/2024-01-24-competition-sign-up-form.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+Teams have been diligently building their robots for 6 months to match their wits against other teams at this grand finale, - and we need **you** to make it happen! 
+
+We try hard to make this event as exciting as possible. Not only for the competitors, but also the volunteers, too. We'll ensure you have a fulfilling and memorable weekend, without causing too much stress. 
+
+Please share this sign-up form with all of your friends and family. The more people we have sign up, the smoother it will run.
```
Comment (prophile) [inline SR2024/2024-01-24-competition-sign-up-form.md:16]: and don't forget to comment and subscribe either

**Review by prophile — APPROVED** [review]

File: SR2024/2024-01-24-competition-sign-up-form.md (hunk 1 above)
Comment (prophile) [reply SR2024/2024-01-24-competition-sign-up-form.md:20]: (this one is the blocker!)

**Comment by RealOrangeOne** [comment]
This got sent

**Pull request merged**